apps/snitch_api/lib/snitch_api_web/controllers/orders_controller.ex

```diff
@@ -0,0 +1,22 @@
+defmodule SnitchApiWeb.OrdersController do
```

Comment:
I used rails way of using plural names for controllers. I did that while setting routes. As I see routes are always plural. So to match with that I used plural controller names. 

But I guess this might confuse a lot of folks in elixir community. So will revert that. 👍